packages/vendure-plugin-invoices/src/strategies/local-file-strategy.ts

```diff
@@ -48,11 +47,11 @@ export class LocalFileStrategy implements LocalStorageStrategy {
     return createReadStream(zipFile);
   }
 
-  async streamFile(invoice: InvoiceEntity, res: Response): Promise<ReadStream> {
+  streamFile(invoice: InvoiceEntity, res: Response): Promise<ReadStream> {
     res.set({
       'Content-Type': 'application/pdf',
       'Content-Disposition': `inline; filename="${invoice.invoiceNumber}.pdf"`,
     });
-    return createReadStream(invoice.storageReference);
+    return Promise.resolve(createReadStream(invoice.storageReference));
```

Comment:
Same for this, we should change the method signature of streamFile to also allow returning a non-promise ReadStream